src/platforms/javascript/common/configuration/tree-shaking/index.mdx

```diff
@@ -79,3 +85,46 @@ const nextConfig = {
 For more information on custom webpack configurations in Next.js, see [Custom Webpack Config](https://nextjs.org/docs/api-reference/next.config.js/custom-webpack-config) in the Next.js docs.
 
 </PlatformSection>
+
+## Tree Shaking Default Integrations
+
+By default, the Sentry SDK sets up a list of [default integrations](../integrations/default) that extend your
+SDK functionality. Furthermore, you can also add [additional](../integrations/plugin) or [custom](../integrations/custom)
+integrations to your SDK configuration.
+In case you do not want to include default integrations in your config, you can [disable](../options/#integration-configuration)
+them and add your custom array of integrations.
+However, if you also want to tree shake the unused default integrations, you can do so by creating a `Client` yourself.
+By doing this, you essentially bypass `Sentry.init` which normally creates a `Client` for you.
+
+The following example shows how to create and bind a `Client` which enables tree shaking of unused default integrations:
+
+```javascript {filename:main.js}
+import {
+  BrowserClient,
+  defaultStackParser,
+  makeFetchTransport,
+  Integrations,
+  getCurrentHub,
+} from "@sentry/browser";
+
+const client = new BrowserClient({
+  // all options you normally pass to Sentry.init
+  dsn: "your DSN",
+  // ...
+
+  transport: makeFetchTransport,
+  stackParser: defaultStackParser,
+  // Only the integrations listed here will be used
+  integrations: [
+    new Integrations.Breadcrumbs(),
+    new Integrations.GlobalHandlers(),
+    new Integrations.LinkedErrors(),
+    new Integrations.Dedupe(),
+  ],
+});
+
+getCurrentHub().bindClient(client);
+```
+
+Note that in contrast to `Sentry.init`, the `Client` constructor expects options of type `ClientOptions` instead of `Options`.
+This means that the `ClientOptions.integrations` property is the final array of integrations that will be used.
```

Comment:
Should this be styled as a note?

```suggestion
<Note>

In contrast to `Sentry.init`, the `Client` constructor expects options of type `ClientOptions` instead of `Options`.
This means that the `ClientOptions.integrations` property is the final array of integrations that will be used.

</Note>
```

Reply:
Admittedly, I didn't think about it but yes it could very well become a note. Thanks for suggesting